src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -181,4 +221,339 @@ private Object createObject(String className) {
         }
         return instance;
     }
+
+    /**
+     * Fill fill module-to-package map.
+     */
+    // -@cs[ExecutableStatementCount] splitting in a few methods is not reasonable.
+    // -@cs[MethodLength] splitting in a few methods is not reasonable.
+    // -@cs[JavaNCSS] splitting in a few methods is not reasonable.
+    private static void fillCheckToPackageMap() {
+        MODULE_TO_PACKAGE_MAP.put("AnnotationLocationCheck",
+                "com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck");
+        MODULE_TO_PACKAGE_MAP.put("AnnotationUseStyleCheck",
```

Comment:
Please name collection by its content.
I do not see package and module. There are short names of classes anf fully qualified names.